src/engine/Marabou.cpp

```diff
@@ -102,6 +102,12 @@ void Marabou::prepareInputQuery()
             printf( "Property: None\n" );
 
         printf( "\n" );
+
+        /*
+          Step 3: extract options
+        */
+        unsigned splitThreshold = Options::get()->getInt( Options::SPLIT_THRESHOLD );
```

Comment:
getInt can return negative value. Need to guard that case

Reply:
guarded